nbterm/types.py

```diff
@@ -0,0 +1,4 @@
+import os
+from typing import Union
+
+PathLike = Union[str, os.PathLike]
```

Comment:
Just needed somewhere to put this to avoid an import cycle between `notebook.py` and `format.py`. Not sure what your preference might be on how to break that cycle - pulling it out like this was my first thought.

Reply:
Personally, I like forcing people to use `Path`s, but for user-facing code it can be convenient to allow strings or `Path`s. Depends on whether you'd like to force people using nbterm programmatically to import and build `Path`s or whether you'll let them use strings. Up to you, I'd say :)